components/ProgressView/src/MDCProgressView.m

```diff
@@ -254,6 +371,16 @@ - (void)announceAccessibilityValueChange {
   }
 }
 
+- (void)startAnimating {
+  [self startAnimatingBar];
+  _isAnimating = YES;
```

Comment:
Set the ivar first in case `startAnimatingBar` or some side effect of it decides it wants to reset `_isAnimating`.